Library/Homebrew/utils/shared_audits.rb

```diff
@@ -70,28 +70,20 @@ def github_release(user, repo, tag, formula: nil, cask: nil)
   def gitlab_repo_data(user, repo)
     @gitlab_repo_data ||= {}
     @gitlab_repo_data["#{user}/#{repo}"] ||= begin
-      out, _, status= curl_output("--request", "GET", "https://gitlab.com/api/v4/projects/#{user}%2F#{repo}")
-      return unless status.success?
-
-      JSON.parse(out)
+      out, _, status = curl_output("https://gitlab.com/api/v4/projects/#{user}%2F#{repo}")
+      JSON.parse(out) if status.success?
     end
-
-    @gitlab_repo_data["#{user}/#{repo}"]
   end
 
   def gitlab_release_data(user, repo, tag)
     id = "#{user}/#{repo}/#{tag}"
     @gitlab_release_data ||= {}
     @gitlab_release_data[id] ||= begin
-      out, _, status= curl_output(
+      out, _, status = curl_output(
         "https://gitlab.com/api/v4/projects/#{user}%2F#{repo}/releases/#{tag}", "--fail"
       )
-      return unless status.success?
-
-      JSON.parse(out)
+      JSON.parse(out) if status.success?
```

Comment:
This will now cache `nil`, while the previous version didn't since it returned when `curl` failed.